.github/workflows/model_ci.yml

```diff
@@ -21,9 +21,9 @@ jobs:
     - name: Train model
       run: |
         rasa train
-    - name: Run Through Test Stories
-      run: |
-        rasa test --stories test_stories/stories.md --fail-on-prediction-errors
+    # - name: Run Through Test Stories
+    #   run: |
+    #     rasa test --stories test_stories/stories.md --fail-on-prediction-errors
```

Comment:
@amn41 Just to be sure that you saw this before merging, I commented out these because of E2E format incompatibility and GH action failing because of it.

Reply:
thanks. what's incompatible?